### PR TITLE
Update README.md.

### PR DESCRIPTION
**Pull request by donraab**
Signed-off-by: Donald Raab <Donald.Raab@bnymellon.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,91 @@
...
+
+
+## Learn Eclipse Collections
+
+* The [Eclipse Collections Katas](https://github.com/eclipse/eclipse-collections-kata), a fun way to help you learn idiomatic Eclipse Collections usage.
+* The [Eclipse Collections Reference Guide](https://github.com/eclipse/eclipse-collections/blob/master/docs/guide.md), an overview of the extensive features available in the framework.
+* The [Javadoc](https://www.eclipse.org/collections/javadoc/9.2.0/overview-summary.html) 
+* [API Design of Eclipse Collections](https://dzone.com/articles/jvm-advent-calendar-api-design-of-eclipse-collecti)
+* [Ten Reasons to use Eclipse Collections](https://medium.com/oracledevs/ten-reasons-to-use-eclipse-collections-91593104af9d)
```
Comment (motlin) [inline README.md]: I don't think this post fits thematically with learning Eclipse Collections.

What order do people actually go in these days? I figure the list should be something like.
* A "little" kata like the Pet kata.
* The company kata.
* The reference guide?

I'm not even sure about the reference guide. I don't think most people read it. Javadoc doesn't belong on the list.

File: README.md [hunk 2]
```diff
@@ -1,30 +1,91 @@
...
+![logo](artwork/eclipse-collections-logo.png) 
+
+
+## [Eclipse Collections](http://www.eclipse.org/collections)
+##### [中文](https://www.eclipse.org/collections/cn/index.html), [Deutsch](https://www.eclipse.org/collections/de/index.html), [Français](https://www.eclipse.org/collections/fr/index.html), [日本語](https://www.eclipse.org/collections/ja/index.html), [Português-Brasil](https://www.eclipse.org/collections/pt-br/index.html), [Русский](https://www.eclipse.org/collections/ru/index.html)
+Eclipse Collections is a collections library for Java with a rich, functional and fluent API. The library provides optimized List, Set and Map implementations and has data structures not found in the JDK including Bags, Multimaps and BiMaps. There is support for primitive versions of Lists, Sets, Bags, Stacks and Maps for all primitive types. There are Mutable and Immutable versions of all container types. The library has support for eager, lazy, and parallel iteration patterns, with eager APIs available directly on the containers. The iteration protocol was inspired by the Smalltalk collection framework, and the collections are compatible with the Java Collection Framework types.
```
Comment (motlin) [inline README.md]: In the interest of conciseness, we can remove feature lists and put them later.

File: README.md [hunk 3]
```diff
@@ -1,30 +1,91 @@
...
+* The [Eclipse Collections Katas](https://github.com/eclipse/eclipse-collections-kata), a fun way to help you learn idiomatic Eclipse Collections usage.
+* The [Eclipse Collections Reference Guide](https://github.com/eclipse/eclipse-collections/blob/master/docs/guide.md), an overview of the extensive features available in the framework.
+* The [Javadoc](https://www.eclipse.org/collections/javadoc/9.2.0/overview-summary.html) 
+* [API Design of Eclipse Collections](https://dzone.com/articles/jvm-advent-calendar-api-design-of-eclipse-collecti)
+* [Ten Reasons to use Eclipse Collections](https://medium.com/oracledevs/ten-reasons-to-use-eclipse-collections-91593104af9d)
+
+
+## How to Contribute
+
+We welcome contributions! We accept contributions via pull requests here in GitHub. Please see [How To Contribute](CONTRIBUTING.md) to get started.
```
Comment (motlin) [inline README.md]: The 99% use-case is using the library without contributing. This belongs at the end.

File: README.md (hunk 1 above)
Comment (donraab) [reply README.md]: Updated

File: README.md [hunk 4]
```diff
@@ -1,30 +1,70 @@
...
+```
 
-The framework is actively being developed, with a growing number of contributors, at the Eclipse Foundation.  The current roadmap for the framework is available [here](https://github.com/eclipse/eclipse-collections/wiki/Roadmap).
+### Jackson Support
 
-Learn Eclipse Collections
-------------------------
-The [Eclipse Collections Reference Guide](https://github.com/eclipse/eclipse-collections/blob/master/docs/guide.md) is a great way to get an overview of the extensive features available in the framework.
+Need support for serializing/deserializing Eclipse Collections types using the popular Jackson library?
+* https://mvnrepository.com/artifact/com.fasterxml.jackson.datatype/jackson-datatype-eclipse-collections
```
[the remark this answers is not in the document]
Comment (donraab) [reply README.md]: Removed

File: README.md [hunk 5]
```diff
@@ -82,129 +122,21 @@ Assert.assertEquals("Smith, Watson, Williams", lastNames.makeString());
 Eclipse Collections has a [lazy API](http://www.eclipse.org/collections/javadoc/9.2.0/org/eclipse/collections/api/LazyIterable.html) as well, which is available by calling the method asLazy().  The method collect will now return a LazyIterable.  The LazyIterable that is returned does not evaluate anything until the call to a terminal method is made.  In this case, the call to makeString() will force the LazyIterable to collect the last names. 
 
 ```java
-ImmutableList<Person> people = Lists.immutable.with(person1, person2, person3);
```
[the remark this answers is not in the document]
Comment (donraab) [reply README.md]: It's duplicate.  The list of people could be either MutableList or ImmutableList.  Both would work.

File: README.md [hunk 6]
```diff
@@ -82,129 +122,21 @@ Assert.assertEquals("Smith, Watson, Williams", lastNames.makeString());
...
-* Implements many high-level iteration patterns (select, reject, collect, inject into, etc.) on "humane" container interfaces which are extensions of the JDK interfaces
-* Provides a consistent mechanism for iterating over Collections, Arrays, Maps, and Strings
-* Provides replacements for ArrayList, HashSet, and HashMap optimized for performance and memory usage
-* Adds new containers including Bag, Interval, Multimap, BiMap, and immutable versions of all types
-* Adds primitive containers for Lists, Sets, Bags, Stacks and Maps for all primitive Java types (boolean, byte, char, short, int, float, long, double)
-* Supports Mutable, Immutable, Synchronized, Unmodifiable and MultiReader Collections
-* Supports eager evaluation with APIs directly on containers and lazy evaluation via a method called asLazy()
-* Encapsulates a lot of the structural complexity of parallel iteration in a parallel API and parallel utility classes 
-* Provides adapters and optimized utility classes for iterating over JDK Collection types with a fluent and friendly API 
-* Has been under active development since 2005 and is a mature library
```
[the remark this answers is not in the document]
Comment (donraab) [reply README.md]: Made the intro more brief.  If you'd like, provide a shorter bulleted list in a PR.

File: README.md (hunk 2 above)
Comment (donraab) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (donraab) [reply README.md]: Moved

File: README.md [hunk 7]
```diff
@@ -82,129 +122,21 @@ Assert.assertEquals("Smith, Watson, Williams", lastNames.makeString());
...
+## Additional information
 
 * Project Website: http://www.eclipse.org/collections
 * Issues: https://github.com/eclipse/eclipse-collections/issues
+* Roadmap: https://github.com/eclipse/eclipse-collections/wiki/Roadmap
 * Wiki: https://github.com/eclipse/eclipse-collections/wiki
 * StackOverflow: http://stackoverflow.com/questions/tagged/eclipse-collections
 * Mailing lists: https://dev.eclipse.org/mailman/listinfo/collections-dev
 * Eclipse PMI: https://projects.eclipse.org/projects/technology.collections
 * Forum: https://www.eclipse.org/forums/index.php?t=thread&frm_id=329
```
[the remark this answers is not in the document]
Comment (donraab) [reply README.md]: Updated

**Pull request merged**